### PR TITLE
add basic container scaffold using bootstrap grids

### PR DESCRIPTION
**Pull request by develra**
This will render strangely on smaller devices - we might want to talk about using grids.

### CODE REVIEW

File: app/index.jade [hunk 1]
```diff
@@ -8,10 +8,13 @@ head
     include ./boilerplate/browsehappy.jade
 
     .container
-      include ./components/header/header.jade
-      include ./components/data-viewer/data-viewer.jade
-      include ./components/template-viewer/template-viewer.jade
-      include ./components/visualizer/visualizer.jade
-      include ./components/footer/footer.jade
+      .row
```
Comment (ianks) [inline app/index.jade:11]: What are the purpose of these rows?

**Comment by develra** [comment]
It is a quick way to make the site render in a reasonable way without using padding. This will get changed when we figure out the header/footer height and fix it to the top + padding.

**Comment by ianks** [comment]
Ok sounds good @develra

**Comment by ianks** [comment]
:+1:

**Comment by ianks** [comment]
:+1:

**Comment by wannabeCitizen** [comment]
Sweet!  Looks like you added in all the changes we were messing with plus got around some of that boilerplate hassle we were experiencing.  :+1:

**Comment by wannabeCitizen** [comment]
Michael, go ahead and merge it yourself, we have all the thumbs up and we'll stick to Ian's recommendation of merging your own requests

**Pull request merged**